packages/trader/test/UniswapV3Trader.ts

```diff
@@ -0,0 +1,427 @@
+import { web3, assert } from "hardhat";
+
+const { toWei, fromWei } = web3.utils;
+
+const { getTruffleContract } = require("@uma/core");
+
+const {
+  encodePriceSqrt,
+  getTickFromPrice,
+  getCurrentPrice,
+  encodePath,
+  computePoolAddress,
+  FeeAmount,
+  createContractObjectFromJson,
+  replaceLibraryBindingReferenceInArtitifact
+} = require("@uma/common");
+
+// Script to test
+const { RangeTrader } = require("../src/RangeTrader");
+
+// Helper scripts
+const { createExchangeAdapter } = require("../src/exchange-adapters/CreateExchangeAdapter");
```

Comment:
Same comments as elsewhere: we should import typescript files as `import X from Y;`. Otherwise, I think typescript has more trouble with it. Note: this may not work for core or common, but I think it's worth a try since they technically have type declarations.